Wordle.Api/Wordle.Api/Services/WordEditorService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Services;
+
+public class WordEditorService(WordleDbContext Db)
+{
+    public WordleDbContext Db { get; set; } = Db;
+
+    public async Task DeleteWord(string Word)
+    {
+        Word? word = await Db.Words.FirstOrDefaultAsync(word => word.Text == Word);
+
+        if(word is not null)
+        {
+            Db.Words.Remove(word);
+            await Db.SaveChangesAsync();
+        }
+
+    }
+
+    public async Task UpdateWord(WordDto wordToEdit)
+    {
+        Word? word = await Db.Words.FirstOrDefaultAsync(word => word.Text == wordToEdit.Word);
+
+        if(word is not null)
+        {
+            word.IsCommonWord = wordToEdit.IsCommonWord;
+
+            await Db.SaveChangesAsync();
+        }
+
+    }
+
+    public async Task AddWord(WordDto wordToAdd)
```

Comment:
Word editor services all look great!

Reply:
Thank you 
![a3f75e84b9542727230fc237ccc4e8d7](https://github.com/IntelliTect-Samples/EWU-CSCD379-2024-Spring/assets/114778753/37d70d0d-9801-44c3-a788-4e8e37d3c6ac)